tools/downloader/converter.py

```diff
@@ -139,6 +177,10 @@ def convert(reporter, model):
         (output_dir / model.subdirectory).mkdir(parents=True, exist_ok=True)
 
         if not run_pre_convert(reporter, model, output_dir, args):
+            try:
+                telemetry.send_error('Pre-conversion script failed.')
```

Comment:
should it be extended with model name (just for clarification which script failed)?